docs/plugins/plugin-script.asciidoc

```diff
@@ -279,3 +285,53 @@ The default location of the `plugins` directory depends on which package you ins
 * {ref}/zip-windows.html#windows-layout[Directory layout of Windows `.zip` archives]
 * {ref}/deb.html#deb-layout[Directory layout of Debian package]
 * {ref}/rpm.html#rpm-layout[Directory layout of RPM]
+
+
+[discrete]
+[[manage-plugins-using-configuration-file]]
+=== Managing plugins using a configuration file
+
+[IMPORTANT]
+. Docker only
+=====================
+At present, this feature is only available when running {es} using our
+Docker images. Other distributions of {es} will not start if they find a
```

Comment:
I don't think we need "at present." Users can assume that.

I also wasn't sure what you meant by "our Docker images." If you mean the official ones, I'd link there so it's a bit more explicit.
```suggestion
This feature is only available for https://www.docker.elastic.co/[official {es}
Docker images]. Other {es} distributions will not start with a
```